src/components/Filters/FilterChecklist.vue

```diff
@@ -134,24 +134,21 @@ export default {
       this.licenseExplanationVisible = false
     },
     isDisabled(e) {
+      const licenses = this.$store.state.filters.licenses
       if (this.$props.filterType === 'licenseTypes') {
-        const nc = this.$store.state.filters.licenses.filter((item) =>
-          item.code.includes('nc')
-        )
-        const nd = this.$store.state.filters.licenses.filter((item) =>
-          item.code.includes('nd')
-        )
+        const nc = licenses.filter((item) => item.code.includes('nc'))
+        const nd = licenses.filter((item) => item.code.includes('nd'))
         return (
           (e.code === 'commercial' && nc.some((li) => li.checked)) ||
           (e.code === 'modification' && nd.some((li) => li.checked))
         )
       }
-
+      const licenseTypes = this.$store.state.filters.licenseTypes
```

Comment:
Using this makes `licenseTypes` non-reactive. It's not recommended but I don't think it wil cause any problems.

Reply:
I refactored the code to avoid this problem . Thank you for pointing it out!